lib/overcommit/installer.rb

```diff
@@ -118,6 +125,32 @@ def can_replace_file?(file)
         overcommit_hook?(file)
     end
 
+    def preserve_old_hooks
+      return unless File.directory?(hooks_path)
+
+      ensure_directory(old_hooks_path)
+      Overcommit::Utils.supported_hook_types.each do |hook_type|
```

Comment:
That's fair, although from my perspective it feels like Overcommit should be an "all-or-nothing" buy in. Having said that, I guess it doesn't hurt to keep this behavior.